portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,50 +46,79 @@ public void init(){
 
     messages = new ArrayList<>();
     messages.add("Hello! How are you");
-    //messages.add("You got this!");
-    //messages.add("Nice to meet you.");
+
+    allComments = new ArrayList<>();
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = names.get(0);
 
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment");
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
-      String whatSaid = (String) entity.getProperty("comment");
-      String mood = (String) entity.getProperty("mood");
-      long timestamp = (Long) entity.getProperty("timestamp");
+      String whatSaid = (String) entity.getProperty("Comment");
+      String mood = (String) entity.getProperty("Mood");
+      //long timestamp = (Long) entity.getProperty("timestamp");
       
-      Comment comment = new Comment(id, whatSaid, mood, timestamp);
-      //messages.add(comment);
+      Comment newComment = new Comment(id, whatSaid, mood);
+      allComments.add(newComment);
     }
-    
-    String json = convertToJsonUsingGson(messages);
+    response.getWriter().println(allComments);
+
+
+    String json = convertToJsonUsingGson(allComments);
     response.setContentType("application/json;");
     response.getWriter().println(json);
+
+    private LinkedHashMap<Integer, Integer> bigfootSightings = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/pit---april-2018-scheduled-traffic-report-reformatted.csv"));
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
```

Comment:
(suggestion only) I would use a CSV library like https://commons.apache.org/proper/commons-csv/ or OpenCSV.  This code would break if for example there were a string entry which contained a comma.